code/modules/mining/machine_processing.dm

```diff
@@ -404,17 +404,14 @@
 			continue
 
 		var/obj/item/stack/ore/O = A
-		if(!O.material)
+		if(!O.materials)
```

Comment:
This fallback code doesn't give you credits.

Reply:
not necessary, gooneores don't give credits